package.json

```diff
@@ -51,12 +51,10 @@
   "private": true,
   "preferGlobal": false,
   "devDependencies": {
-    "babel-preset-env": "1.6.1",
-    "babel-preset-es2015": "6.24.1",
-    "babel-preset-es2017": "6.24.1",
-    "babelify": "8.0.0",
+    "babel-preset-env": "^1.6.1",
+    "babelify": "^8.0.0",
```

Comment:
still required by https://github.com/ethereumjs/ethereumjs-util/blob/cf511cc0a16ebd3a026b99b46b33adcfa13217c9/package.json#L20-L31 (was fixed in ethereumjs-block so not as many now)

Reply:
I am afraid I need more context – I am not sure what is happening here :)
Is `^` still required, or `babel-preset-env` ?

Reply:
sorry, ^ is not required, `babel-preset-env` and `babelify` are still required here because `etheriumjs-util` has their project setup incorrectly